docs/installation/instalation-and-requirements.md

```diff
@@ -0,0 +1,233 @@
+# Installation and Requirements
+
+Implementing of this library inside your existing project is not that trivial.
+
+That is why we created a boilerplate repository that incorporates this library. To see the detailed implementation and how it works in action check the [Eightshift Boilerplate repo](https://github.com/infinum/eightshift-boilerplate).
+
+**In order to incorporate this lib inside your project you need to have:**
+
+1. [Node.js](https://nodejs.org/en/) and [Composer](https://getcomposer.org/) installed on your system
+
+**Implementation process:**
+
+1. Install the frontend library and eightshift libs
+
+    ```shell script
+    npm install @eightshift/frontend-libs
+    composer require infinum/eightshift-libs
+    ```
+
+2. If you have an existing Webpack config, look at how the config is set up in [Eightshift Boilerplate repo](https://github.com/infinum/eightshift-boilerplate/blob/develop/webpack.config.js) and merge any changes you have.
+
+The `eightshift-libs` composer package contains a lot of useful abstract classes, interfaces and dependency injection container implementation that you can use in your project. 
+
+### Examples
+
+Inside your theme's `functions.php` you'd register a main class like this
```

Comment:
This whole section below doesn't really make sense for `frontend-libs`. You're talking about registering a main class and adding new classes (`php` stuff) which is https://github.com/infinum/eightshift-libs territory. If anything, this should / could be added there.

Reply:
Yeah, the issue is that blocks just won't work the way it's described here unless you install `eightshift-libs`. It's kinda a hard dependency. 

You can use it for the mixins and stuff I guess, we could maybe add some notification above about this :+1:

Reply:
> Yeah, the issue is that blocks just won't work the way it's described here unless you install eightshift-libs. It's kinda a hard dependency.
> 
> You can use it for the mixins and stuff I guess, we could maybe add some notification above about this 👍

Yeah I'm quite aware, tho that should probably be at the top of `README.md`.

I'm realising the issue that we have:

This whole part should go to https://github.com/infinum/eightshift-libs (for which we also need docs). But it's also important for this lib. And ideally we need all of it in one place.

So we probably need to think about having a separate DocPress repo which will hold the stuff for both `eightshift-libs` and `eightshift-frontend-libs` (rather than having each repository have it's own DocPress OR even worse, this repository holding docs for both).

For now I guess it's fine but I'd rather if we solved this sooner rather than later.